src/components/seeds/database/index.tsx

```diff
@@ -478,29 +481,16 @@ export default function Database(props: DatabaseProps): JSX.Element {
           horizontal: 'left',
         }}
       >
-        <Box padding={2}>
-          <Box>
-            <IconButton
-              onClick={() => onDownloadReport()}
-              sx={{ ':hover': { background: 'none' } }}
-              id='download-report'
-            >
-              <Icon name='export' />
-              <Typography sx={{ color: '#136BD3', paddingLeft: 1 }}>{strings.EXPORT_RECORDS}</Typography>
-            </IconButton>
-          </Box>
-          <Box>
-            <IconButton
-              onClick={() => onOpenEditColumnsModal()}
-              sx={{ ':hover': { background: 'none' } }}
-              id='edit-columns'
-            >
-              <Icon name='iconColumns' />
-
-              <Typography sx={{ color: '#136BD3', paddingLeft: 1 }}>{strings.CUSTOMIZE_TABLE_COLUMNS}</Typography>
-            </IconButton>
-          </Box>
-        </Box>
+        <MenuList sx={{ padding: theme.spacing(2, 0) }}>
```

Comment:
remove spacing here?

Reply:
Removing parent padding and adding to children looks like this:

<img width="333" alt="Screen Shot 2022-09-28 at 16 17 25" src="https://user-images.githubusercontent.com/5919083/192869736-3286d9d0-3083-4cd8-a133-c75bd7429220.png">

Is that correct?

Reply:
hmm..my mistake. looks like your original version is consistent with our menus

<img width="277" alt="Screen Shot 2022-09-28 at 12 22 53 PM" src="https://user-images.githubusercontent.com/1865174/192870590-75d78f61-bb1e-4d99-ba21-e50a6fc55fe6.png">